plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -121,16 +121,18 @@ private static IFile materialize(IProject project, AppEngineProjectConfig config
     }
 
     if (isStandardProject) {
-      createChildFile("appengine-web.xml", AppEngineTemplateUtility.APPENGINE_WEB_XML_TEMPLATE,
+      createChildFile("appengine-web.xml",  //$NON-NLS-1$
+          AppEngineTemplateUtility.APPENGINE_WEB_XML_TEMPLATE,
           webinf, subMonitor.newChild(5), properties);
     } else {
-      copyChildFile("app.yaml", webinf, subMonitor.newChild(5));
+      IFolder appengine = createChildFolder("appengine", main, subMonitor.newChild(5)); //$NON-NLS-1$
+      copyChildFile("app.yaml", appengine, subMonitor.newChild(5));
```

Comment:
Seems like we don't even need this. Someone else is creating this file already. Need to find who and where.